pubnub-core/pubnub-core-api/src/iosMain/kotlin/com/pubnub/kmp/Downloadable.ios.kt

```diff
@@ -0,0 +1,3 @@
+package com.pubnub.kmp
+
+actual abstract class Downloadable
```

Comment:
What is a reason to make `Downloadable` abstract class instead of an interface?

Reply:
because on JVM we're aliasing this to InputStream which is an `abstract class` and they must match
otherwise yes, this would be an interface